rpc/block_test.go

```diff
@@ -616,6 +858,10 @@ func TestBlockTransactionCount(t *testing.T) {
 				BlockID:       WithBlockNumber(30000),
 				ExpectedCount: 4,
 			},
+			{
+				BlockID:       WithBlockNumber(52959),
```

Comment:
Would you mind updating the assertions used in the test here? We should e able to clean it up quite a bit to use require.Equal() and require.NoError() etc instead of all the if err, t.Fatal() code. Thanks

Reply:
I'll get on it🫡